common/include/pcl/PCLPointCloud2.h

```diff
@@ -18,9 +18,7 @@ namespace pcl
 
   struct PCL_EXPORTS PCLPointCloud2
   {
-    PCLPointCloud2 () : height (0), width (0), 
-                     is_bigendian (false), point_step (0), row_step (0),
-                     is_dense (false)
+    PCLPointCloud2 ()
```

Comment:
Why not using in-class initialization for `is_bigendian` as well and getting rid of the explicit constructor entirely?

Reply:
Please have a look at #3440

Reply:
Hmm. Let me close this PR, so you can create the new child PR.